modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -161,28 +161,39 @@ const opMakeRowHeader = (grid: Structs.RowCells[], detail: Structs.DetailExt, co
 };
 
 const opMakeRowsHeader = (initialGrid: Structs.RowCells[], details: Structs.DetailExt[], comparator: CompElm, genWrappers: GeneratorsTransform) => {
-  const rows = uniqueRows(details);
-
-  const replacer = (currentGrid: Structs.RowCells[], row: Structs.DetailExt) => TransformOperations.replaceRow(currentGrid, row.row, comparator, genWrappers.replaceOrInit);
+  const replacer = (currentGrid: Structs.RowCells[], row: Structs.DetailExt) =>
```

Comment:
I'd call it `rowReplacer` (and `columnReplacer` below) just cause they have the same name and do different things (I know they are in different scopes but...)

Reply:
Not related to this change (I've only applied some better formatting)